test/agent_test.go

```diff
@@ -77,8 +85,12 @@ func TestAgents(t *testing.T) {
 			return "room job not assigned"
 		}
 
-		if ac3.participantJobs.Load()+ac4.participantJobs.Load() != 1 {
-			return fmt.Sprintf("participant jobs not assigned, ac3: %d, ac4: %d", ac3.participantJobs.Load(), ac4.participantJobs.Load())
+		if ac3.publisherJobs.Load()+ac4.publisherJobs.Load() != 1 {
```

Comment:
why is this `!= 1`? Is only one of those publishers getting assigned a job? Is that because of some capacity limitation in this test?

Reply:
this checks the number of times the worker successfully registers with the server

Reply:
Got it. I was puzzled by why the sum of those two publishers is 1. The check below checks that the sum is 2. So, was puzzled.